packetery/src/Packetery/Module/Options/Page.php

```diff
@@ -211,7 +201,13 @@ public function render(): void {
 
 		$latteParams['apiPasswordLink'] = trim( $this->latte_engine->renderToString( PACKETERY_PLUGIN_DIR . '/template/options/help-block-link.latte', [ 'href' => 'https://client.packeta.com/support' ] ) );
 		$latteParams['senderLink']      = trim( $this->latte_engine->renderToString( PACKETERY_PLUGIN_DIR . '/template/options/help-block-link.latte', [ 'href' => 'https://client.packeta.com/senders' ] ) );
-		$latteParams['exportLink']      = $this->httpRequest->getUrl()->withQueryParameter( 'action', 'export-settings' )->getRelativeUrl();
+		$latteParams['exportLink']      = add_query_arg(
+			[
+				'page'   => 'packeta-options',
+				'action' => 'export-settings',
```

Comment:
export-settings se také používá na více místech,  také by si to zasloužilo nějakou konstantu.